### PR TITLE
SAK-44216 Messages: Advanced search doesn't work finding by label on other languages

### PR DESCRIPTION
**Pull request by maurercw**
The commits from October must have been leftover when I was accidentally working in a different branch, but they have already been merged to master.

### CODE REVIEW

File: msgcntr/messageforums-component-impl/src/test/java/org/sakaiproject/component/app/messageforums/MsgcntrTestConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+    @Bean(name = "org.sakaiproject.springframework.orm.hibernate.GlobalSessionFactory")
+    public SessionFactory sessionFactory() throws IOException {
+        DataSource dataSource = dataSource();
+        LocalSessionFactoryBuilder sfb = new LocalSessionFactoryBuilder(dataSource);
+        StandardServiceRegistryBuilder srb = sfb.getStandardServiceRegistryBuilder();
+        srb.applySetting(org.hibernate.cfg.Environment.DATASOURCE, dataSource);
+        srb.applySettings(hibernateProperties());
+        StandardServiceRegistry sr = srb.build();
+        sr.getService(MutableIdentifierGeneratorFactoryInitiator.INSTANCE.getServiceInitiated())
+                .register("uuid2", AssignableUUIDGenerator.class);
```
Comment (ern) [inline msgcntr/messageforums-component-impl/src/test/java/org/sakaiproject/component/app/messageforums/MsgcntrTestConfiguration.java:114]: Is this needed? If not can ellminate the srb pieces.

**Review by ern — APPROVED** [review]
Thanks for the unit tests!

File: msgcntr/messageforums-component-impl/src/test/java/org/sakaiproject/component/app/messageforums/MsgcntrTestConfiguration.java (hunk 1 above)
Comment (maurercw) [reply msgcntr/messageforums-component-impl/src/test/java/org/sakaiproject/component/app/messageforums/MsgcntrTestConfiguration.java:114]: Not sure.  I just followed an example from somewhere else!  I'll take a look...

Comment (maurercw) [reply msgcntr/messageforums-component-impl/src/test/java/org/sakaiproject/component/app/messageforums/MsgcntrTestConfiguration.java]: Was able to remove some stuff!

**Pull request merged**